crypto/crypto.go

```diff
@@ -77,15 +77,15 @@ func Decrypt(base64EncodedCiphertext string, key []byte) []byte {
 }
 
 // DeriveKey derives a 32 byte encryption key from a password and identifier.
-func DeriveKey(password, identifier []byte) []byte {
-	derivedKey, _ := scrypt.Key(password, identifier, 1<<18, 8, 1, 32)
+func DeriveKey(password, identifier []byte, costFactor map[string]int) []byte {
+	derivedKey, _ := scrypt.Key(password, identifier, 1<<uint(costFactor["N"]), costFactor["r"], costFactor["p"], 32)
```

Comment:
🤔 Well there's a default set in `pocket.go` so the only way the value would be zero would be if the user manually and explicitly sets it to zero. Not sure if I should disallow that. Scrypt would throw an error and quit anyways so is it really needed?